src/components/inline-inputs/inline-inputs.spec.tsx

```diff
@@ -203,6 +203,21 @@ describe("Inline Inputs", () => {
     });
   });
 
+  describe("when providing the labelAlign prop", () => {
+    beforeEach(() => {
+      wrapper = render({ label: "foo", labelAlign: "left", labelWidth: 30 });
+    });
+
+    it("sets the inline label to the provided alignemnt", () => {
```

Comment:
**question**: could we do an` it.each` here to check `labelAlign` being `"left"`/`"right"` and the css value of justify-content being `"flex-start"`/`"flex-end"`, just like its done in `inline-inputs.pw.tsx`?

Reply:
I could do but I personally just wanted to check that the prop is passing through as expected for unit test coverage and then the css checks are reserved for playwright. If it's felt that we should be css checking here then I'll make that change

Reply:
Yeah, that's fair, I think the playwright check is enough actually in retrospect

Reply:
Traditionally we'd have also tested the CSS in the unit tests but I think there was a discussion around not doing this when we move to RTL? I'm OK doing it the way you have and you seem to have full coverage too.